### PR TITLE
Developer friendly error for params class method

### PR DESCRIPTION
**Pull request by tak1n**
This will most likely not be a problem in a full-blown hanami application, as I could imagine the overall `hanami` gem would also install `hanami/validations`. But if using `hanami/controller` standalone this will raise a more developer-friendly error to instruct to install `hanami/validations`.

It is already somewhat documented in the README (explicit require for `hanami/validations`), but a more helpful runtime error could not harm.

Feel free to point out some way how to test this or whether the test can be skipped as it is such a minor thing.

EDIT: Found the isolation specs :)

### CODE REVIEW

File: lib/hanami/action.rb [hunk 1]
```diff
@@ -214,6 +214,19 @@ def self.accepted_formats
...
+    #
+    # Raises a developer friendly error to include `hanami/validations`.
+    #
+    # @raise [NoMethodError]
+    #
+    # @api private
+    # @since 2.0.0
+    def self.params(klass = nil, &blk)
+      raise NoMethodError,
+            "To use Hanami::Action.params include hanami/validations gem in your Gemfile"
```
Comment (cllns) [inline lib/hanami/action.rb:227]:
```suggestion
            "To use `params`, please add 'hanami/validations' gem to your Gemfile"
```

**Review by cllns — COMMENTED** [review]
Great idea! I made a minor suggested change to the language, what do you think of it?

**Comment by tak1n** [comment]
> Great idea! I made a minor suggested change to the language, what do you think of it?

Sounds way better, thanks. ✌🏽

**Comment by timriley** [comment]
Nice improvement, thank you!

**Pull request merged**